app/test/admin/exported_api_sync_test.dart

```diff
@@ -0,0 +1,194 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:_pub_shared/data/admin_api.dart';
+import 'package:gcloud/storage.dart';
+import 'package:pub_dev/shared/configuration.dart';
+import 'package:pub_dev/shared/storage.dart';
+import 'package:pub_dev/shared/versions.dart';
+import 'package:test/test.dart';
+
+import '../shared/test_models.dart';
+import '../shared/test_services.dart';
+
+void main() {
+  group('Exported API sync admin action', () {
+    Future<void> _sync({
+      List<String>? packages,
+      bool forceWrite = false,
+    }) async {
+      final api = createPubApiClient(authToken: siteAdminToken);
+      await api.adminInvokeAction(
+        'exported-api-sync',
+        AdminInvokeActionArguments(arguments: {
+          'packages': packages?.join(' ') ?? 'ALL',
+          if (forceWrite) 'force-write': 'true',
+        }),
+      );
+    }
+
+    Future<Map<String, dynamic>> _list() async {
```

Comment:
```suggestion
    /// Return map of all objects in the Exported API bucket.
    ///
    /// Returns a map from object name to JSON on the form:
    /// ```js
    /// {
    ///   'updated': '<date-time>',
    ///   'metadata': {
    ///     'contentType': '<contentType>',
    ///     'custom': {...},
    ///   },
    ///   'length': <length>,
    ///   'bytes': [...],
    /// }
    /// ```
    Future<Map<String, dynamic>> listExportedApi() async {
```

This will make the tests a lot easier to read.